### DISCUSSION TITLE
The version 15.3.0 does not work as a new installation

### DISCUSSION THREAD

**Discussion by martin8310** [jomjol/AI-on-the-edge-device#2724] [General]
### The Problem

The version 15.3.0 does not work as a new installation. Error 0x10
14.0.3 works fine. 

Hardware: ESP32-CAM-CH340

### Version

14.0.3

### Logfile

```shell
0x00000010 SD folder or file presence check failed
```

**Comment by caco3** [comment]
What is the content of sour SD-card? Did you set it up beforehand?

**Comment by contentpirate** [comment]
Same problem here

Tried to use different SD cards, different ESP32. I used the web installer, copied all the files to the SD card including changing the wifi.ini file. It didn't work, so I copied the config.ini and the wlan.ini manually to the SD card again. No success.

Logfile: https://pastebin.com/B6YZJQva

**Comment by caco3** [comment]
> copied all the files to the SD card

Can you describe this a bit better?
What did you copy?
Can you show a screenshot of `http://<IP>/fileserver/#` ?
The log sais you dont have a `logs` folder: `E (8438) LOGFILE: Can't open log file /sdcard/log/message/[..]`

**Comment by contentpirate** [comment]
> > copied all the files to the SD card
> Can you describe this a bit better? What did you copy?


I downloaded "AI-on-the-edge-device__manual-setup__v15.3.0.zip", unzipped and copied the content of "sd-card.zip" on to the sd card. Then, I changed the wifi credentials in the wifi.ini file and put it in the esp32.


> Can you show a screenshot of `http://<IP>/fileserver/#` ? The log sais you dont have a `logs` folder: `E (8438) LOGFILE: Can't open log file /sdcard/log/message/[..]`

Sure!

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/146078371/659820f5-0157-4a6a-987c-003643d2b7e4)

This is the page that asks me upload the remote_setup.zip - I tried multiple times by uploading "AI-on-the-edge-device__remote-setup__v15.3.0.zip". But different to the instruction that it would be coming in three steps, the second step never came up after uploading the AI-on-the-edge-device__remote-setup__v15.3.0.zip and therefore finishing step 1, but it directly jumps to step 3.

I tried to recreate it for this thread here, but now even the upload doesn't work.

Logfile: https://pastebin.com/WNhCauZA

What might be wrong here? Hm...


I tried once again setting up the SD card manually like above just with a different SD card, and then it appeared to do something else - it apparently connected to the wifi and received an IP address, but I wasn't able to access the web interface.

Logfile: https://pastebin.com/nbgY0iBt

**Comment by caco3** [comment]
```
I (15329) MAIN: Initialization completed successfully
I (15369) ov2640: Set PLL: clk_2x: 0, clk_div: 0, pclk_auto: 0, pclk_div: 8
E (17019) sdmmc_cmd: sdmmc_read_sectors_dma: sdmmc_send_cmd returned 0x109
E (17029) diskio_sdmmc: sdmmc_read_blocks failed (265)
Guru Meditation Error: Core  0 panic'ed (LoadProhibited). Exception was unhandled.
```

There is an issue with your SD card!
Can you try with a different one?

**Reply by contentpirate** [reply]
Strange! I tried with the Intenso MSDHC4G MicroSDHC-Speicherkarte 8GB card - is there any recommended SD card?

**Reply by MichaelOi** [reply]
The "Version.txt" file in the html folder is missing. When you add an empty text file "Version.txt" the ESP starts up, but I get a message (wrong firmware version).

**Reply by contentpirate** [reply]
It's strange, because I double-checked if theres's a "version.txt" in the folder. There was, and since I found a different thread with the same issue (with the missing empty version.txt), I simply deleted the content to have an empty version.txt. Didn't work.
I ordered all new ESP32 and 4GB micro SD cards to try now.

**Reply by caco3** [reply]
please do another setup using the OTA function to fix the missing files.
And don't mess around on the card manually.

**Comment by contentpirate** [comment]
Hello hello, I tried with another ESP32 and the same SD cards, but it didn't work.
The non-working SD card was an Intenso MSDHC4G MicroSDHC 8GB card - so I ordered an Intenso MSDHC4G MicroSDHC **card with only 4 GB**.

It arrived today, I used the same way of manual SC card configuration by copying the content of sd-card.zip and adding the wifi credentials.
It works now!

Apparently, the ESP32 or AI on the edge is **very picky with SD card sizes**. 4GB card works, 8GB card doesn't.

Log FYI: https://pastebin.com/RWvcdQkR

Thanks a lot to everyone!

**Reply by caco3** [reply]
its the ESP and their framework which is picky, we can't do much against it.
